src/rubymotion-apple.h

```diff
@@ -29,6 +29,7 @@ VALUE rb_uint2big(VALUE);
 long rb_num2int(VALUE);
 #define FIX2INT(x) ((int)FIX2LONG(x))
 #define NUM2INT(x) (FIXNUM_P(x) ? FIX2INT(x) : rb_num2int((VALUE)x))
+#define SSIZET2NUM(v) LONG2NUM(v)
```

Comment:
We can. I used `SSIZET2NUM` to remain consistent with the [Ruby C API](https://docs.ruby-lang.org/en/trunk/extension_rdoc.html), but defer to you. Let me know either way!